SPECS/ca-certificates/ca-certificates.spec

```diff
@@ -45,7 +45,7 @@ Name:           ca-certificates
 # When updating, "Epoch, "Version", AND "Release" tags must be updated in the "prebuilt-ca-certificates*" packages as well.
 Epoch:          1
 Version:        2.0.0
-Release:        13%{?dist}
+Release:        14%{?dist}
```

Comment:
Why are we targeting `fasttrack/2.0` for this specific change instead of `main` ?